src/day8/re_frame/localstorage.cljs

```diff
@@ -0,0 +1,16 @@
+(ns day8.re-frame.trace.localstorage
+  (:require [goog.storage.Storage :as Storage]
+            [goog.storage.mechanism.HTML5LocalStorage :as html5localstore]))
+
+(defn- storage []
+  (let [mech (goog.storage.mechanism.HTML5LocalStorage.)
+        store (goog.storage.Storage. mech)]
+    store))
+
+(defn get! [key]
+  (let [store (storage)]
+    (cljs.reader/read-string (.get store key))))
```

Comment:
It's good practice to require `cljs.reader` to show that you use it further down, and so that if it is no longer automatically required by CLJS core, that this code will still work.